src/transformers/models/deta/configuration_deta.py

```diff
@@ -109,6 +109,10 @@ class DetaConfig(PretrainedConfig):
             based on the predictions from the previous layer.
         focal_alpha (`float`, *optional*, defaults to 0.25):
             Alpha parameter in the focal loss.
+        assign_first_stage (`bool`, *optional*, defaults to `True`):
+            Whether to assign each prediction i to the highest overlapping ground truth object if the overlap is larger than a threshold τ.
```

Comment:
Does this threshold τ correspond to a value in the configuration file? If so, that should be referenced here instead of  the greek letter

Reply:
@amyeroberts this is not the value related to configuration. Since it is confusing I changed to iou value with 0.7!